File: src/main/java/com/james/Ticket.java
package com.james;

public class Ticket {
    private int origin;
    private int destination;
    private int price;

    public Ticket() {
    }

    public Ticket(int origin, int destination, int price) {
        this.origin = origin;
        this.destination = destination;
        this.price = price;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getOrigin() {
        return origin;
    }

    public void setOrigin(int origin) {
        this.origin = origin;
    }

    public int getDestination() {
        return destination;
    }

    public void setDestination(int destination) {
        this.destination = destination;
    }


}

File: src/main/kotlin/com/james/Ticket.kt
package com.kotlin

import kotlin.random.Random

data class Ticket(val origin: Int, val destination: Int, val price: Int)

fun main(args: Array<String>) {
    var ticket = Ticket(12, 34, 56)
    val s = "aaaass"
    println(s.validate())
    println((1..10).random())

}

//自訂義擴充原本class沒有的method
fun String.validate(): Boolean = this.length >= 6

fun IntRange.random(): Int {//自訂義擴充原本class沒有的method
    val r = Random.nextInt(endInclusive - start) + start
    return r

}